.github/workflows/ci-core-reusable.yml

```diff
@@ -233,8 +263,18 @@ jobs:
 
       - name: Start services
         run: |
-          docker-compose -f docker-compose-runner.yml pull
-          docker-compose -f docker-compose-runner.yml up --build -d geth zk postgres
+          mkdir -p ./volumes/postgres
+          mkdir -p ./volumes/geth/keystore
+          mkdir -p ./volumes/prysm/beacon
+          mkdir -p ./volumes/prysm/validator
+          cp ./docker/prysm/config.yml ./volumes/prysm/config.yml
+          cp ./docker/geth/jwtsecret ./volumes/geth/jwtsecret
+          cp ./docker/geth/password.sec ./volumes/geth/password.sec
+          cp ./docker/geth/keystore/UTC--2019-04-06T21-13-27.692266000Z--8a91dc2d28b689474298d91899f0c1baf62cb85b ./volumes/geth/keystore/
+          docker-compose -f docker-compose-local-genesis.yml up
+          docker-compose -f docker-compose-local-genesis.yml rm -f
+          docker-compose --profile runner up -d --wait
```

Comment:
could we create a shortcut for it?

Reply:
Yep, I just don't know the best approach. A script?

Reply:
shell script.. like `ci_run`

Reply:
Done